spec/models/project_group_spec.rb

```diff
@@ -55,4 +55,148 @@
       expect(group.descendant_step_count).to eq group.children.size
     end
   end
+
```

Comment:
I have duplicated the project_step_spec logic here. If this is how we want to go forward, I can extract into a shared spec.